Dockerfile

```diff
@@ -9,11 +9,13 @@ VOLUME /root/.m2
 COPY . /usr/src/app
 WORKDIR /usr/src/app
 
-RUN mvn clean package -Dcheckstyle.skip
+RUN mvn clean install -Dcheckstyle.failOnViolation=false -B
 
 EXPOSE 8080
 # Cache maven dependencies
 VOLUME /root/.m2
 
 ENTRYPOINT [ "mvn" ]
-CMD [ "exec:java", "-Dcheckstyle.skip" ]
+
+# Execution without checkstyle and tests
+CMD [ "exec:java", "-Dcheckstyle.skip", "-DskipTests=true", "-B" ]
```

Comment:
End file with newline pls

Reply:
Why?

Reply:
Because that’s how the POSIX standard defines a line:

A sequence of zero or more non- <newline> characters plus a terminating <newline> character.